src/ui/control/attribution_control.js

```diff
@@ -69,7 +70,7 @@ class AttributionControl {
         if (!this._editLink) this._editLink = this._container.querySelector('.mapbox-improve-map');
         if (this._editLink) {
             const center = this._map.getCenter();
-            const styleParams = (this.styleOwner && this.styleId) ? `?owner=${this.styleOwner}&id=${this.styleId}` : '';
+            const styleParams = (this.styleOwner && this.styleId) ? config.ACCESS_TOKEN ? `?owner=${this.styleOwner}&id=${this.styleId}&access_token=${config.ACCESS_TOKEN}` : `?owner=${this.styleOwner}&id=${this.styleId}` : '';
```

Comment:
@anandthakker I think it should https://github.com/mapbox/mapbox-gl-js/blob/78fc9c415a2d5bc71d9d10d8bdcf90c98387846a/src/index.js#L50